test/LightAccount.t.sol

```diff
@@ -282,10 +284,10 @@ contract LightAccountTest is Test {
             keccak256(
                 abi.encodePacked(
                     type(LightAccountFactory).creationCode,
-                    bytes32(uint256(uint160(0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789)))
+                    bytes32(uint256(uint160(0x0000000071727De22E5E9d8BAf0edAc6f37da032)))
```

Comment:
We can replace this line with just `abi.encode(<EntryPoint address>)` to avoid dealing with the type conversions.

Reply:
Ack, will fix up the stack.